spec.go

```diff
@@ -393,9 +393,18 @@ func createCgroupConfig(name string, spec *specs.Spec) (*configs.Cgroup, error)
 		}
 		if r.CPU.Quota != nil {
 			c.Resources.CpuQuota = int64(*r.CPU.Quota)
+			// This is a limit set in the kernel
+			// See https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt
+			if c.Resources.CpuQuota < 1000 {
+				return nil, fmt.Errorf("CPU Quota cannot be less than 1ms i.e. 1000")
+			}
 		}
 		if r.CPU.Period != nil {
 			c.Resources.CpuPeriod = int64(*r.CPU.Period)
+			// Similar to CpuQuota limit
+			if c.Resources.CpuPeriod < 1000 {
+				return nil, fmt.Errorf("CPU Period cannot be less than 1ms i.e. 1000")
+			}
```

Comment:
For period, if we check the minimal, I think we need to check maximal as well, cause it has 1s upper limit.